src/index.js

```diff
@@ -0,0 +1,149 @@
+import './style.css';
+import '../node_modules/@fortawesome/fontawesome-free/js/all.js';
+
+const todoHTML = document.querySelector('.todo_list');
+const todoInput = document.querySelector('.todo_input');
+const clearAllBtn = document.querySelector('.clear_btn');
+
+const todoList = {
+  todoTasks: [],
+};
+
+const setLocalStore = (todoTasks) => {
+  localStorage.setItem('tasks', JSON.stringify(todoTasks));
+};
+
+const addTask = (desctiption, completed = false, index, todoList) => {
+  const task = {
+    desctiption,
+    completed,
+    index,
+  };
+  todoList.todoTasks.push(task);
+  setLocalStore(todoList.todoTasks);
+};
+
+const removeTask = (i, todoList) => {
+  todoList.todoTasks.splice(i, 1);
+  todoList.todoTasks.forEach((task) => {
+    task.index = i + 1;
+  });
+  setLocalStore(todoList.todoTasks);
+};
```

Comment:
the removeTask function is running an issue after removing a task you update all the elements with the same index i+1, you can solve this by decreasing the current index by 1 for each task starting from the removed index.
![ismailtodolist](https://user-images.githubusercontent.com/59707859/154974410-aaf8568e-72e5-4b81-8f93-a98593dbe21f.PNG)